compiler/options.nim

```diff
@@ -37,7 +37,9 @@ type                          # please make sure we have under 32 options
                               # evaluation
     optPatterns,              # en/disable pattern matching
     optMemTracker,
-    optHotCodeReloading
+    optHotCodeReloading,
+    optSourceMap
```

Comment:
I think ``optSourceMap`` should be a global option instead.

Reply:
ok